docs/reference/cli.md

```diff
@@ -2,812 +2,835 @@
 title: CLI Reference
```

Comment:
This is generated so it shouldn't be manually changed

Reply:
Oh, how is it generated? I can update that instead

Reply:
It's generated by `internal/docgen`